pkgs/by-name/gk/gk-cli/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  coreutils,
+  installShellFiles,
+  makeWrapper,
+}:
+
+let
+  version = "2.1.0";
+in
+stdenv.mkDerivation {
+  pname = "gk-cli";
+  inherit version;
+
+  src = fetchzip {
+    url = "https://github.com/gitkraken/gk-cli/releases/download/v${version}/gk_${version}_Linux_x86_64.zip";
+    sha256 = "sha256-BZO8MVFxIXXB+9v4yB9XcZ0E68xOIUL7Mot7PZ+5M1U=";
```

Use of `hash` over `sha256` is preferred as the former provides an easier path to change the hash algorithm in the future
```suggestion
    hash = "sha256-BZO8MVFxIXXB+9v4yB9XcZ0E68xOIUL7Mot7PZ+5M1U=";
```